packages/atoms/src/Icon/Icon.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+    Canvas,
+    ImageSVG,
+    useSVG,
+    Group,
+    Paint,
+    BlendColor,
+    usePaintRef,
+} from '@shopify/react-native-skia';
+import { IconType, iconTypes } from './iconTypes';
+import { useNativeStyles } from '@trezor/styles';
+import { Color } from '@trezor/theme';
+
+const iconSizes = {
+    small: 16,
+    big: 24,
+} as const;
```

Comment:
Why not use enum? You wouldn't need to use this const assertion and it's much cleaner in my opinion. You get the narrowed type by default.

Reply:
I usually used enums with data models. When it's a react component I prefer the JS way and simply defining an object. For example, I would make the Trezor model number an `enum` because it's a part of the global data mode. I hope I explain it well x) Also I think that enum in it's simplest form (when the value does not matter and stated explicitly) is the perfect case for enums. 


```
enum TransactionType {
  Fiat,
  Crypto,
  Candy
}
```

When you use string enums or define the numbers it gets tricky. 

But also when you are syncing with a backend enums are handy, since it's the same type and value at the same time in one place in the codebase.

Reply:
Also u don't need to import the enum to pass the prop value, it's just a string and better autocomplete experience